src/main/java/com/incognia/IncogniaAPI.java

```diff
@@ -166,69 +149,55 @@ public TransactionAssessment registerLogin(String installationId, String account
    * <pre>{@code
    * IncogniaAPI api = new IncogniaAPI("client-id", "client-secret", Region.BR);
    * try {
-   *      TransactionAssessment assessment = api.registerLogin("installation-id", "account-id", "external-id");
+   *     RegisterLoginRequest loginRequest = RegisterLoginRequest.builder()
+   *         .installationId("installation-id")
+   *         .accountId("account-id")
+   *         .externalId("external-id")
+   *         .evaluateTransaction(true)
+   *         .build();
+   *      TransactionAssessment assessment = api.registerLogin(loginRequest);
    * } catch (IncogniaAPIException e) {
    *      //Some api error happened (invalid data, invalid credentials)
    * } catch (IncogniaException e) {
    *      //Something unexpected happened
    * }
    * }</pre>
    *
-   * @param installationId the <a
-   *     href="https://docs.incognia.com/learning/concepts/identifiers/installation-id">installation
-   *     id</a>
-   * @param accountId the account id
-   * @param externalId client-owned transaction identifier
+   * @param request the {@link RegisterLoginRequest} model with the properties we need to make the
+   *     assessment
    * @return the assessment for the login
    * @throws IncogniaAPIException in case of api errors
    * @throws IncogniaException in case of unexpected errors
    */
-  public TransactionAssessment registerLogin(
-      String installationId, String accountId, String externalId) throws IncogniaException {
-    Asserts.assertNotEmpty(installationId, "installation id");
-    Asserts.assertNotEmpty(accountId, "account id");
+  public TransactionAssessment registerLogin(RegisterLoginRequest request)
+      throws IncogniaException {
+    RegisterLoginRequest loginRequest =
+        RegisterLoginRequest.builder()
+            .installationId("installation-id")
+            .accountId("account-id")
+            .externalId("external-id")
+            .evaluateTransaction(true)
+            .build();
```

Comment:
Should this be here? Looks like it isn't being used :eyes:

Reply:
you're right :eyes:

Reply:
removed, thanks, nice catch :D